kitchen.yml

```diff
@@ -155,6 +155,7 @@ suites:
       state_top:
         base:
           '*':
+            - TEMPLATE._mapdata
```

Comment:
Will need this line below for the `centos6` suite as well.

---

In fact, the includes/excludes need to be fixed here as well, which I'll probably do in a separate PR first.  So with that fix but without this line below, we have it failing:

* https://travis-ci.org/github/myii/template-formula/builds/723150877

I'll get this working and then mention the further changes that are needed here.

---

And while I've been adding the rest of the comments here, this is the test run with the actual diffs:

* https://travis-ci.org/github/myii/template-formula/builds/723154962

Reply:
OK, I've fixed it:

* https://travis-ci.org/github/myii/template-formula/builds/723159868

However, there were a lot of minor changes required, that aren't relevant to this PR.  Let me submit a PR to fix those first and then this PR can be rebased on top.

Reply:
Fix provided: #204.